CONTRIBUTORS.md

```diff
@@ -0,0 +1,20 @@
+Abel 'Akronix' Serrano Juste
```

Comment:
Since we aren't using Markdown at all in this file, should we just rename it to CONTRIBUTORS (sans extension) so it renders as plain text?